app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/migration/NameIdOrgController.scala

```diff
@@ -188,7 +188,7 @@ class NameIDOrgController @Inject() (
     Map(
       CdsOrganisationType.CompanyId                     -> UtrConfiguration("Corporate Body", displayMode = RegisteredCompanyDM),
       CdsOrganisationType.PartnershipId                 -> UtrConfiguration("Partnership", displayMode = PartnershipDM),
-      CdsOrganisationType.LimitedLiabilityPartnershipId -> UtrConfiguration("LLP", displayMode = PartnershipDM),
+      CdsOrganisationType.LimitedLiabilityPartnershipId -> UtrConfiguration("LLP", displayMode = PartnershipLLP),
       CdsOrganisationType.CharityPublicBodyNotForProfitId -> UtrConfiguration(
         "Unincorporated Body",
         displayMode = OrganisationModeDM,
```

Comment:
I'm not sure why we have this as a map of String->UtrConfiguration (which is a confusing term, this seems to be viewmodel information). Is there a reason this isn't a Map[CdsOrganisationType, UtrConfiguration] with the UtrConfigurations defined in their own file (and maybe this as a method on the UtrConfiguration object?)

Reply:
We'll stick to the changes we discussed.